include/deal.II/numerics/vector_tools.h

```diff
@@ -24,8 +24,14 @@
 #include <deal.II/base/std_cxx11/function.h>
 #include <deal.II/dofs/function_map.h>
 #include <deal.II/dofs/dof_handler.h>
+#include <deal.II/dofs/dof_tools.h>
 #include <deal.II/hp/dof_handler.h>
 #include <deal.II/hp/mapping_collection.h>
+#include <deal.II/lac/solver_cg.h>
+#include <deal.II/lac/precondition.h>
+#include <deal.II/matrix_free/matrix_free.h>
```

Comment:
No, I should have been more precise: ...which I am doing right now. Compiling really takes forever :unamused: (as noted in #3388).